plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -46,10 +47,7 @@
     protected void setup(Binder binder)
     {
         configBinder(binder).bindConfig(GlueHiveMetastoreConfig.class);
-
-        newOptionalBinder(binder, GlueColumnStatisticsProvider.class)
-                .setDefault().to(DisabledGlueColumnStatisticsProvider.class).in(Scopes.SINGLETON);
-
+        configBinder(binder).bindConfig(HiveConfig.class);
```

Comment:
It feels to me that you should not need this one as it is bind in `HIveModule` which should be always present.

Reply:
Do we have a test that verify this ?

Reply:
I do not believe we have. AFAIK (and I could not find any) we have no proper product-tests coverage for Trino run agains AWS.
cc: @findepi

Reply:
Correct, we have only non-product tests, like `TestHiveGlueMetastore`
this is tracked by https://github.com/trinodb/trino/issues/5426